cmd/harmony/main.go

```diff
@@ -116,8 +115,10 @@ var (
 	enableMemProfiling = flag.Bool("enableMemProfiling", false, "Enable memsize logging.")
 	enableGC           = flag.Bool("enableGC", true, "Enable calling garbage collector manually .")
 	blsKeyFile         = flag.String("blskey_file", "", "The encrypted file of bls serialized private key by passphrase.")
-	blsPass            = flag.String("blspass", "", "The file containing passphrase to decrypt the encrypted bls file.")
-	blsPassphrase      string
+	blsFolder          = flag.String("blsfolder", "", "The encrypted file of bls serialized private key by passphrase.")
```

Comment:
description not updated.

Reply:
also, please add a message saying that all blskey has to be in the same shard, also share the same pass.

Reply:
`blsFolder          = flag.String("blsfolder", ".hmy/blskeys", "The folder that stores the bls keys; same blspass is used to decrypt all bls keys; all bls keys mapped to same shard")`